changelog/dmd.named-arguments.dd

```diff
@@ -0,0 +1,33 @@
+Named arguments have been implemented
+
+[DIP1030: Named Arguments](https://www.dlang.org/dips/1030) has been partially implemented.
+In a function call expression, an argument can new be prefixed by the identifier of the parameter it binds to.
+
+---
+struct S{}
+struct T{}
+
+string f(T t, int i,     S s) => "A";
+string f(S s, int i = 0, T t) => "B";
+
+void main()
+{
+    S s;  T t;  int i;
+
+    // positional
+    assert(f(t, i, s) == "A");
+    assert(f(s, i, t) == "B");
+    // f(s, t); // error, neither A nor B match
+    // f(t, s); // error, neither A nor B match
+
+    // named arguments
+    // f(s: s, t: t, i: i); // error, ambiguous
+    assert(f(s: s, t: t) == "B");
+    assert(f(t: t, s: s) == "B");
+    assert(f(t: t, i, s: s) == "A");
+    assert(f(s: s, t: t, i) == "A");
```

Comment:
The single letter parameter names and argument names make this really hard to follow. I'd suggest, since there are no "refs" here, just use literals. And change the parameter names to something like `tVal`, `intVal`, etc.

The changelog is to help understand the feature. This kind of brevity/abstraction can be fine in a spec document.